src/components/data-management/ProjectMenu.jsx

```diff
@@ -54,6 +60,64 @@ const ProjectMenu = (props) => {
     });
     return canLaunch;
   };
+
+  const getProjectHash = (project) => {
+    const samplesToHash = project.samples.map((sampleUuid) => ({
+      name: samples[sampleUuid].name,
+      metadata: samples[sampleUuid].metadata,
+    }));
+
+    return hash.MD5(samplesToHash);
+  };
+
+  // Initialize hash for each project
```

Comment:
Why is the hash initialized for each project here? Why not calculate it in the reducer when project is loaded/modified or something?

Reply:
Hmm.. Good idea. I'll move it to the reducers,